iOS/UI/Swipeable Input/SwipeableInputAccessoryView.swift

```diff
@@ -292,117 +292,115 @@ class SwipeableInputAccessoryView: View, AttachmentViewControllerDelegate, UIGes
+        if didFinish {
+            self.selectionFeedback.impactOccurred()
+            self.previewView?.removeFromSuperview()
 
-        if let point = self.interactiveStartingPoint {
-            startingPoint = point
-        } else {
-            // Initial location
-            startingPoint = pan.location(in: nil)
-            self.interactiveStartingPoint = startingPoint
-        }
-
-        let totalOffset: CGFloat = clamp(self.halfHeight, InputAccessoryView.preferredHeight, InputAccessoryView.maxHeight * 0.5)
-        var diff = (startingPoint.y - currentLocation.y)
-        diff -= totalOffset
-        var progress = diff / 100
-        progress = clamp(progress, 0.0, 1.0)
+            if let object = self.sendableObject {
+                if progress > 0.99 {
+                    self.delegate.swipeableInputAccessory(self, didConfirmReply: object)
+                } else {
+                    self.delegate.swipeableInputAccessory(self, didConfirm: object)
+                }
 
-        switch pan.state {
-        case .possible:
-            break
-        case .began:
-            self.previewView = PreviewMessageView()
-            self.panDidBegin()
-            self.previewView?.backgroundView.alpha = 0.0
-            self.addSubview(self.previewView!)
-            self.previewView?.frame = self.inputContainerView.frame
-            self.previewView?.layoutNow()
-            let top = self.top - totalOffset
-
-            self.previewAnimator = UIViewPropertyAnimator(duration: Theme.animationDuration,
-                                                          curve: .easeInOut,
-                                                          animations: nil)
-            self.previewAnimator?.addAnimations {
-                UIView.animateKeyframes(withDuration: 0,
-                                        delay: 0,
-                                        animations: {
-                                            UIView.addKeyframe(withRelativeStartTime: 0,
-                                                               relativeDuration: 0.3,
-                                                               animations: {
-                                                                self.attachmentView.alpha = 0
-                                                                self.textView.alpha = 0
-                                                                self.previewView?.backgroundView.alpha = 1
-                                            })
-
-                                            UIView.addKeyframe(withRelativeStartTime: 0,
-                                                               relativeDuration: 1,
-                                                               animations: {
-                                                                self.previewView?.top = top
-                                                                self.setNeedsLayout()
-                                            })
-
```

Comment:
Because you can drag horizontally and vertically, I removed the animator approach and instead move the view manually.